src/js/profile/mobile/widget/Calendar.js

```diff
@@ -40,19 +44,23 @@
 				events = ns.event,
 				days = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"],
 				fullNameMonth = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
-				today = new Date(),
-				todayYear = today.getFullYear(),
-				todayMonth = today.getMonth() + 1,
-				defaultToday = today.getUTCDate(),
+				dateData = new Date(),
+				todayYear = dateData.getFullYear(),
+				todayMonth = dateData.getMonth() + 1,
+				defaultToday = dateData.getUTCDate(),
+				loadTodayFlag = true,
+				activeMonth,
+				fixMonth,
 				switchElement,
+				arrowElement,
 				selectDay,
 
 				Calendar = function () {
 					this.options = utilsObject.merge({}, defaultOptions);
 				},
 
 				defaultOptions = {
-					pastSelection: false
+					pastSelection: "false"
```

Comment:
Description from commit:
"If the value of "data-past-selection" is true, the past date cannot be selected."

I'm understanding this option different. I think this should be opposite. When option is "true" then user can select past date.

You should describe this option by jsDoc
```
				/**
				 * Options for widget.
				 * @property {Object} options
				 * @property {boolean} [options.pastSelection=true] description
				 * @member ns.widget.mobile.Calendar
				 */
				this.options = {
					pastSelection: true
				};
```